test/TEST-45-REPART/testsuite.sh

```diff
@@ -0,0 +1,121 @@
+#!/bin/bash
+set -ex
+
+systemd-analyze log-level debug
+
+# Check if repart is installed, and if it isn't bail out early instead of failing
+test -x /usr/bin/systemd-repart || exit 0
```

Comment:
Almost there - the integration tests logic checks for the `/testok` file (and its contents) created when the test finishes successfully (like below on line 119), which in this case is missing - that's why CI still complains

Reply:
ok, thanks for tracking this down! you are a lifesaver!

(hopefully) fixed now.